src/plugins_types.c

```diff
@@ -44,12 +44,14 @@
  *
  * @param [in,out] err_code return code. if thre is not enough memory for allocating error message then this code change to LY_EMEM.
  * @param [in] err_vecode Validity error code in case of LY_EVALID error code.
+ * @param [in] path   	Path to the node causing the error.
```

Comment:
inconsistent indentation, keep there just a single space

Reply:
I see that it was changed in a later commit, but I don't see any reason to do not to rebase the fix back into the commit it introduced the spaces. And even after the later change, there are still more spaces.